docs/conf.py

```diff
@@ -1,19 +1,26 @@
 # -*- coding: utf-8 -*-
 #
-# stravalib documentation build configuration file, created by
-# sphinx-quickstart on Wed Dec 25 15:30:40 2013.
+# Configuration file for the Sphinx documentation builder.
 #
-# This file is execfile()d with the current directory set to its
-# containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
 
-import sys
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
 import os
+import sys
+sys.path.insert(0, os.path.abspath('../'))
```

Comment:
path maybe windows?
you can use `os.sep` or something else

Reply:
@yihong0618 can you please clarify what you are looking for / suggesting here? thank you. this should run across operating systems.